tests/ci_visibility/util.py

```diff
@@ -118,3 +120,76 @@ def _mock_upload_git_metadata(obj, **kwargs):
         "ddtrace.internal.ci_visibility.recorder._do_request", side_effect=NotImplementedError
     ):
         yield
+
+
+_PYTEST_SNAPSHOT_GITLAB_CI_ENV_VARS = {
+    "GITLAB_CI": "true",
+    "CI_COMMIT_AUTHOR": "TestFirst TestLast <First.Last@testtest.com>",
+    "CI_COMMIT_MESSAGE": "test commit message",
+    "CI_COMMIT_REF_NAME": "test.brancn/test_name",
+    "CI_COMMIT_SHA": "c165eb71ef833b752783b5268f21521fd16f812a",
+    "CI_COMMIT_TIMESTAMP": "2024-09-10T10:11:13+01:00",
+    "CI_COMMIT_TAG": "v1.0.0",
+    "CI_JOB_ID": "633358062",
+    "CI_JOB_NAME": "test-job",
+    "CI_JOB_NAME_SLUG": "test-job-slug",
+    "CI_JOB_STAGE": "test-stage",
+    "CI_JOB_URL": "https://test.test.io/Test/test-test/test-test/-/jobs/633358062",
+    "CI_PIPELINE_ID": "43949931",
+    "CI_PIPELINE_IID": "14726",
+    "CI_PIPELINE_URL": "https://test.†est.io/Test/test-†est/test-test/-/pipelines/43949931",
+    "CI_PROJECT_PATH": "Test/test-test/test-project-path",
+    "CI_PROJECT_PATH_SLUG": "test-test-test-test-test-project-path",
+    "CI_PROJECT_URL": "https://test.test.io/Test/test-test/test-test",
+    "CI_REPOSITORY_URL": "https://test.test.io/Test/test-test/test-test.git",
+    "CI_RUNNER_ID": "14727097",
+    "CI_RUNNER_TAGS": '["runner:test-test-test-test"]',
+}
+
+
+def _get_default_os_env_vars():
+    os_env = os.environ
+
+    os_env_keys = {
+        "PATH",
+        "PYTHONPATH",
+        "DD_TRACE_AGENT_URL",
+        "DD_AGENT_PORT",
+        "DD_TRACE_AGENT_PORT",
+        "DD_AGENT_HOST",
+        "DD_TRACE_AGENT_HOSTNAME",
+    }
+
+    return {key: os_env.get(key, "") for key in os_env_keys if key in os_env}
```

Comment:
Since you already check that the key is present, we don't need `get()`.
```suggestion
    return {key: os_env[key] for key in os_env_keys if key in os_env}
```